packages/core/src/config/loader.ts

```diff
@@ -1,11 +1,12 @@
 import { createRequire } from 'node:module';
 import * as path from 'node:path';
+import * as fs from 'node:fs';
 import SilentError from 'silent-error';
 import { GlintConfig } from './config.js';
 import { GlintConfigInput } from '@glint/core/config-types';
 import type * as TS from 'typescript';
 
-const require = createRequire(import.meta.url);
+export const require = createRequire(import.meta.url);
```

Comment:
why export this? 
conventionally, we really only want to use createRequire in the file it's used in, since the resolution rules are specific to the import.meta.url

Reply:
ah, it's for mocking in tests.

can you add a block-comment with something like:
```js
/**
 * @private 
 *
 * Only exported for testing purposes. Do not import.
 */
```

Reply:
**Done**